ddsreplayer/test/blackbox/mcap/resources/config_file_end_time.yaml

```diff
@@ -1,6 +1,9 @@
 builtin-topics:
   - name: /dds/topic
     type: HelloWorld
+
+topics:
+  - name: /dds/topic
     qos:
       reliability: true       # Use QoS RELIABLE
```

Comment:
Please add a note in documentation (all documentations actually) warning the user of what might occur when only setting durability qos.

Reply:
Apart from this general Fast-DDS fact, which is not bad to have, I was referring more to the concrete case in which only one of them is set via specs/participant_qos/manual_topics while the other gets set through discovery.